<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelPages</class>
 <widget class="QWidget" name="PanelPages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="SectionContainer" name="sectionParentPages">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title" stdset="0">
       <string>Parent Pages</string>
      </property>
      <widget class="QWidget" name="layoutParentPages">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="PageList" name="listParentPages"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QToolButton" name="buttonAddParent">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonImportParent">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonDuplicateParent">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="TrashBinWidget" name="labelDeleteParentPage"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="SectionContainer" name="sectionDocumentPages">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="title" stdset="0">
          <string>Document Pages</string>
         </property>
         <widget class="QWidget" name="layoutDocumentPages">
          <property name="font">
           <font>
            <bold>false</bold>
           </font>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0">
           <property name="spacing">
            <number>8</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>8</number>
             </property>
             <property name="rightMargin">
              <number>8</number>
             </property>
             <item>
              <widget class="FormWidget" name="formWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="label">
                <string>Document Layout</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QComboBox" name="comboDocumentLayout"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="FormWidget" name="layoutFirstPage">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="label">
                <string>First Page</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="FirstPageComboBox" name="comboFirstPage"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="PageViewer" name="viewDocumentPages" native="true"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>8</number>
             </property>
             <property name="rightMargin">
              <number>8</number>
             </property>
             <item>
              <widget class="QToolButton" name="buttonAddPage">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="buttonImportPage">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="buttonDuplicatePage">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="buttonMovePage">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="TrashBinWidget" name="labelDeletePage"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormWidget</class>
   <extends>QWidget</extends>
   <header>form_widget/form_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SectionContainer</class>
   <extends>QWidget</extends>
   <header>section_container/section_container.h</header>
   <addpagemethod>addPage</addpagemethod>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageList</class>
   <extends>QListWidget</extends>
   <header>page_list/page_list.h</header>
  </customwidget>
  <customwidget>
   <class>FirstPageComboBox</class>
   <extends>QComboBox</extends>
   <header>firstpage_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>PageViewer</class>
   <extends>QWidget</extends>
   <header location="global">page_viewer/page_viewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TrashBinWidget</class>
   <extends>QLabel</extends>
   <header>trashbin_widget/trashbin_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
